devops/Makefile

```diff
@@ -38,10 +38,10 @@ LIBS = $(LIB_DYNAMIC) $(LIB_STATIC)
 # package related variables
 PACKAGE_NAME ?= $(LIBNAME)
 PRE_PACKAGE_GOALS := pre_package $(LIB_DYNAMIC)
-FPM_P_MAINTAINER := Evernym
-FPM_P_URL := https://github.com/evernym/libsovtoken
+FPM_P_MAINTAINER := Sovrin
+FPM_P_URL := https://github.com/sovrin/libsovtoken
```

Comment:
~~`sovrin`~~ -> `sovrin-foundation`

Reply:
@felippeb could you address this concern? So that future build artifacts contain 
correct links